hdfs-dataset-sink/src/main/java/org/springframework/cloud/stream/module/dataset/sink/DatasetSinkProperties.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.dataset.sink;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Used to configure the HDFS Dataset Sink module options that are related to writing using Kite SDK.
+ *
+ * @author Thomas Risberg
+ */
+@SuppressWarnings("unused")
+@ConfigurationProperties
+public class DatasetSinkProperties {
+
+	/**
+	 * The URI to use to access the Hadoop FileSystem.
+	 */
+	private String fsUri;
+
+	/**
+	 * The base directory path where the files will be written in the Hadoop FileSystem.
+	 */
+	private String directory = "/tmp/hdfs-dataset-sink";
```

Comment:
Should this depend on the "stream name"? ie `@Value("${spring.application.name:foo}")`